<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(247, 255, 243);</string>
  </property>
  <widget class="QLabel" name="loginLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string> Логин</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Пароль</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLabel_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Подтвердите пароль</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="loginEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordEdit_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="registerButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 241, 185);</string>
   </property>
   <property name="text">
    <string>Зарегистрироваться</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>400</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(243, 238, 255);</string>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>170</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(229, 229, 229);</string>
   </property>
   <property name="text">
    <string>☼</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showButton_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>210</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(229, 229, 229);</string>
   </property>
   <property name="text">
    <string>☼</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
